tests/utils/pod.go

```diff
@@ -67,6 +67,7 @@ func DeletePod(clientSet *kubernetes.Clientset, pod *k8sv1.Pod, namespace string
 
 // NewPodWithPVC creates a new pod that mounts the given PVC
 func NewPodWithPVC(podName, cmd string, pvc *k8sv1.PersistentVolumeClaim) *k8sv1.Pod {
+	fsGroup := int64(107)
```

Comment:
what exactly is 107?

Reply:
Good point, should have used the constant we have defined for it.